### ISSUE TITLE
Setting the technology has no effect

### ISSUE THREAD

**Issue by oiiiiiiii** [goadesign/model#763]
When creating a relationship between to elements e.g. with `Uses(FCMHandler, "Sends out notifications via", "HTTP")` the technology (here "HTTP") is completely ignored and not being rendered to the arrow.  C4 usually contains those in "[, ]", like "[HTTP]", see https://static.structurizr.com/workspace/76749/diagrams/Containers.png

**Comment by raphael** [comment]
Yeah I'd be happy to merge a PR that added the technology to the link in diagrams!

**Comment by oiiiiiiii** [comment]
Unfortunately I have no idea how to set up this project to that I can't really test a modified version. I took a look at the code though and I think it could be as easy as replacing `graph.addEdge(rel.id, rel.sourceId, rel.destinationId, rel.description, ref.vertices, style)` in the `parseMode.ts` file with something like
```
graph.addEdge(rel.id, rel.sourceId, rel.destinationId, rel.technology ? `${rel.description}\n[${rel.technology}` : rel.description, ref.vertices, style)
```
(I replaced rel.description)
The technology property seems to be implemented everywhere, just not used for the label argument of addEdge(). But again, I didn't test it and maybe it's not as easy as I think, I am not familiar with the codebase...

Sorry for not really contributing the way you had hoped :/
